hwdb/60-evdev.hwdb

```diff
@@ -253,6 +253,13 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pvrThinkPad??40?:*
  EVDEV_ABS_35=::41
  EVDEV_ABS_36=::37
 
+# Lenovo ThinkPad T430
```

Comment:
Don't know about funny spaces, but I typed it in by hand now to get rid of any funny characters.